nix/noninteractive.nix

```diff
@@ -21,6 +21,9 @@
 
   # would pull in nano
   programs.nano.syntaxHighlight = lib.mkForce false;
+  programs.nano.enable = false;
+
+  documentation.man.man-db.enable = false;
```

Comment:
Afaik. the same as `documentation.enable = false;`, which we have set for all installers.

Reply:
During my testing (on nixos-23.11) it turned out, counter-intuitively indeed, that `documentation.man.man-db.enable` is not in the same scope of `documentation.enable`. This may not be the case in later nixos versions, but for 23.11 this was true.

`/misc/documentation.nix` always imports `/misc/man-db.nix` which does not observe `documentation.enable`, but only `documentaion.man.man-db.enable` value.

If I recall correctly, this was another couple of MB of savings.

Reply:
Alright.

Reply:
Also on unstable it seems to help... I cherry-picked this one as well.